dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -284,13 +284,15 @@ public Set<String> getOrderableFields() {
 
   private Predicate<TrackedEntity> getFilter(
       UserDetails user, TrackedEntityQueryParams queryParams) {
+    boolean skipOwnershipCheck = queryParams.getOrgUnitMode() == ALL;
```

Comment:
Users need a special authority to use the mode `ALL` right?

Reply:
yes, but that's already validated in `OperationsParamsValidator`, before we hit the database